src/content/language/isolates.md

```diff
@@ -39,6 +39,12 @@ but here are some more situations where they can be useful:
 - Performing I/O, such as communicating with a database.
 - Handling a large volume of network requests.
 
+Also note a caveat due to a limit to the number of concurrently running isolates:
```

Comment:
I'd rephrase this as

"""
Synchronous blocking communication between isolates.    

If isolates want to communicate synchronously they have to leave the safety of Dart and resort to using C code via FFI to do so. Special care must be taken in this circumstance due to the Dart VM imposing a limit on the number of isolates that can run in parallel. If the number of isolates that synchronously block in FFI calls exceeds that limit no other isolate can run. If those blocking isolates wait on another unscheduled isolate to act, then deadlock can occur. To avoid this situation (preventing other isolates from running & possibly deadlock scenarios) such blocking C code must take care of leaving the current isolate before performing the blocking operation and re-entering the isolate before returning from the FFI call (see ...)
"""

(especially concurrently -> in parallel - we can run thousands of isolates concurrently but not in parallel, i.e. at the same time)

Reply:
This text sounds good, but did you mean for it to replace my version or you thought about adding it?